src/DynamoCoreWpf/Extensions/ViewLoadedParams.cs

```diff
@@ -148,6 +148,19 @@ private void OnSelectionCollectionChanged(object sender, NotifyCollectionChanged
             }
         }
 
+        /// <summary>
+        /// This event is raised when an extension tab is closed and this event 
+        /// is subscired by the Workspace dependency view extension
+        /// </summary>
+        public event Action OnExtensionTabClosed;
```

Comment:
can we make this internal for now? 
Also we should probably put the name of the extension that is being closed here as a parameter so that other extensions can check if they should respond or not...

Reply:
Yes, I forgot to this change this to internal. Changed it now